google/cloud/spanner/database_admin_connection.h

```diff
@@ -174,11 +175,11 @@ class DatabaseAdminConnection {
     /// The name of the database.
     Database database;
     std::string backup_id;
-    std::chrono::system_clock::time_point expire_time;
+    Timestamp expire_time;
     // The backup will contain an externally consistent copy of the database
     // at `version_time`. If `version_time` is not specified, the system will
     // set `version_time` to the `create_time` of the backup.
-    absl::optional<std::chrono::system_clock::time_point> version_time;
+    absl::optional<Timestamp> version_time;
```

Comment:
Changing the types of these fields will be a hard break for customers right away. Maybe we should leave the old fields, and _add_ the new fields. We could mark the old fields deprecated, and indicate that they are ignored. WDYT?

Reply:
Done.

(Aside: It only matters for `expire_time` ... `version_time` is a new field.)